app/views/partials/_nav.html.erb

```diff
@@ -0,0 +1,13 @@
+<nav class="primary_nav">
```

Comment:
should this be in the app layout? Since it's gonna show everywhere

Reply:
Not quite everywhere. Like not the log in page... Maybe better with an if statement, we can discuss